frontend/web/manifest.json

```diff
@@ -0,0 +1,35 @@
+{
+    "name": "frontend",
+    "short_name": "frontend",
```

Comment:
Probably needs whitelabeling aswell. We could also just build this only for the Ehrenamtskarte Bayern app.

In both cases -> New issue about whitelabeling and do that in a separate issue.

Reply:
```suggestion
    "name": "Ehrenamtskarte Bayern",
    "short_name": "Ehrenamt",
```